core/src/database/DatabaseSessionPool.hpp

```diff
@@ -59,7 +59,7 @@ namespace ledger {
                 const std::string &password = ""
             );
 
-            static const int CURRENT_DATABASE_SCHEME_VERSION = 14;
+            static const int CURRENT_DATABASE_SCHEME_VERSION = 15;
```

Comment:
@KhalilBellakrid is going to change that value in a soon-to-be-merged PR too. Which one will have the conflict?

FIGHT! 😆

Reply:
**15** has always been my favourite number, so... 😅